frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -309,17 +309,44 @@
 
         vm.inviteOthers = function(ev, hostTeamId) {
             ev.stopPropagation();
-            // Appending dialog to document.body 
-            var confirm = $mdDialog.prompt()
-                .title('Invite others to this Team')
-                .textContent('Enter the email address of the person')
-                .placeholder('email')
-                .ariaLabel('')
-                .targetEvent(ev)
-                .ok('Send Invite')
-                .cancel('Cancel');
-
-            $mdDialog.show(confirm).then(function(result) {
+            var inviteTemplate = 
```

Comment:
@Obtin I think this isn't the right way of adding a template. We must use a separate file for it and then we can give it's path here. Please let me know if something is unclear.